numba/types/npytypes.py

```diff
@@ -338,6 +338,11 @@ def can_convert_to(self, typingctx, other):
                 and (self.aligned or not other.aligned)):
                 return Conversion.safe
 
+        if self.ndim == 1 and isinstance(other, Sequence):
```

Comment:
I am worried about the implicit conversion of arrays into tuples.  I think it is safer to do this with explicit conversion by introducing a new function.  The implicit conversion will affect overload resolution.